src/TinyHelpers.AspNetCore/Attributes/FileSizeAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+using Microsoft.AspNetCore.Http;
+
+namespace TinyHelpers.AspNetCore.Attributes;
+
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false)]
+public class FileSizeAttribute : ValidationAttribute
+{
+    private readonly int maxFileSizeInBytes;
+
+    public FileSizeAttribute(int maxFileSizeInBytes)
+        : base("{0} size cannot be bigger than {1} Bytes")
```

Comment:
Typo: _Bytes_ invece di _bytes_. Inoltre lo standard dei messaggi di errore di default prevede un formato del tipo  `The {0} field...`